File: questions/Pascal_triangel.py
class Solution:
    def generate(self, numRows):
        """
        :type numRows: int
        :rtype: List[List[int]]
        """
        L=[]
        for x in range(numRows):
            if x is 0:
                L.append([1])
            else:
                tem_list = L[x-1][:]    
                tem_list.append(0)      
                new = []
                for i in range(x+1):    
                    st = tem_list[i] + tem_list[i-1]  
                    new.append(st)
                L.append(new)
        return L 

File: questions/Pascal_triangel.go
func generate(numRows int) [][]int {
   list := make([][] int , numRows)
   switch {
       case numRows == 0:
        list = [] [] int {}
       case numRows == 1:
        list[0] =  [] int {1}
       case numRows > 1 :
        for i := 0;i < numRows;i++{
            if i == 0 || i== 1 {
                list[0] ,list[1] = [] int {1} ,[] int {1,1}
            }else {
            tmp := make([] int ,i+1)  
            for j := 0 ;j < i+1 ;j++{
                if j == 0 || j == i{
                    tmp[0] , tmp[i] = 1,1
                }else{
                    tmp[j] = list[i-1][j-1] + list[i-1][j]
                }
              }  
           list[i] = tmp
            }
          }
        }
    return list 
    }
